engine/swarm/configs.md

```diff
@@ -612,3 +612,53 @@ configuration file.
 
 You have now updated your `nginx` service's configuration without the need to
 rebuild its image.
+
+### Example: Use templated config in Nginx service
+
+To create a configuration in which the content will be generated using a
+template engine, use the `--template-driver` parameter and specify the engine
+name as its argument. The template will be rendered when container is created.
+
+1.  Save the following into a new file `index.html`.
+
+    ```html
+    <html>
+      <head><title>Hello Docker</title></head>
```

Comment:
Do we want to template this as well?

Reply:
Should it be done as a separate environmental variable `TITLE` or reuse `HELLO`?

Reply:
I think we can reuse `HELLO` to keep it simple, but either way is fine; wdyt?